### PR TITLE
[issue-299] allow single line of text in free form text

### LINKED ISSUE #299: Allow single lines during parsing of text values in tag-value format
Fields that currently only allow `"<text>...</text>"` strings should also allow single lines of text (i.e. strings without `"\n"`).

### PR DESCRIPTION
**Pull request by meretp**
fixes #299

Signed-off-by: Meret Behrens <meret.behrens@tngtech.com>

### CODE REVIEW

**Comment by meretp** [comment]
Todo: adapt docstrings

**Comment by meretp** [comment]
> Todo: adapt docstrings

Done, ready for review

File: spdx/parsers/tagvalue.py [hunk 1]
```diff
@@ -1287,6 +1301,14 @@ def p_snippet_lic_comment_1(self, p):
         msg = ERROR_MESSAGES["SNIP_LICS_COMMENT_VALUE"].format(p.lineno(1))
         self.logger.log(msg)
 
+    def p_text_or_line_value_1(self, p):
+        """text_or_line : TEXT"""
+        p[0] = p[1]
+
+    def p_text_or_line_value_2(self, p):
+        """text_or_line : LINE"""
+        p[0] = p[1]
```
Comment (nicoweidner) [inline spdx/parsers/tagvalue.py:1310]: This looks absurd, but seems to be doing the right thing!

**Review by nicoweidner — APPROVED** [review]
Thanks for the fix! I only noticed 2 typos that we may as well fix

File: spdx/parsers/tagvalue.py (hunk 1 above)
Comment (armintaenzertng) [reply spdx/parsers/tagvalue.py:1310]: Yes, in the tag-value lexer, a `TEXT` is a string (possibly multi-line) enclosed in `<text>`, while `LINE` is a single line without the tags. Some properties don't allow `TEXT`, so there has to be a distinction.

File: tests/data/doc_write/tv-simple-plus.tv [hunk 2]
```diff
@@ -16,7 +16,7 @@ PackageVerificationCode: SOME code
 PackageLicenseDeclared: NOASSERTION
 PackageLicenseConcluded: NOASSERTION
 PackageLicenseInfoFromFiles: LGPL-2.1-or-later
-PackageCopyrightText: <text>Some copyrught</text>
+PackageCopyrightText: Some copyrught
```
[the remark this answers is not in the document]
Comment (armintaenzertng) [reply tests/data/doc_write/tv-simple-plus.tv:19]: There are actually 14 occurences of `copyrught` in the project :D

[the remark this answers is not in the document]
Comment (meretp) [reply tests/data/doc_write/tv-simple-plus.tv:19]: If I remember correctly, it was once fixed in a PR, but probably reverted during some merge conflicts or whatever, but I will change this now, it really bothers my inner Monk (not sure if this is the correct translation)..

File: tests/data/doc_write/tv-simple.tv [hunk 3]
```diff
@@ -16,7 +16,7 @@ PackageVerificationCode: SOME code
 PackageLicenseDeclared: NOASSERTION
 PackageLicenseConcluded: NOASSERTION
 PackageLicenseInfoFromFiles: LGPL-2.1-only
-PackageCopyrightText: <text>Some copyrught</text>
+PackageCopyrightText: Some copyrught
```
[the remark this answers is not in the document]
Comment (meretp) [reply tests/data/doc_write/tv-simple.tv]: fixed

**Review by nicoweidner — APPROVED** [review]

**Pull request merged**